src/quo2/foundations/colors.cljs

```diff
@@ -13,9 +13,21 @@
       (let [rgb (string/split value #",")]
         (str (string/join "," (butlast rgb)) "," opacity ")")))))
 
+(defn theme-alpha ([light-color light-opacity dark-color dark-opacity]
```

Comment:
discussing with Pedro in ux and it seems quite common where a community will provide a color and then we have to use that color with different opacity (can be different for dark and light).

Added this help function as it will be a common utility we need.

Reply:
Nothing major or that needs to be done now, but the `theme-alpha` function could be memoized (simplest strategy, just stores in memory without any cache eviction strategy). I think for lower level abstractions that can get called quite a lot, memoization can be very advantageous.